CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# How to contribute
+
+## Rights
+The license (see LICENSE) applies to all contributions.
+
+## Issue Conventions
+When submitting bugs, please filll out the bug report template in full.
```

Comment:
Hi Andrew! Thanks for the great work and sorry for commenting out of nothing...i've just seen by coincidence that here is a little typo ;-)

Reply:
Fixed in 1e0c370. Thanks, @jlandmann!